### PR TITLE
removed dependency on set version of react and react-dom

### PR DESCRIPTION
**Pull request by cwebley**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,9 +32,11 @@
     "babel-preset-es2015": "^6.3.13",
     "happiness": "github:jedwatson/happiness"
   },
+  "peerDependencies": {
+	  "react": "*",
```
Comment (wesleytodd) [inline package.json]: use 2 spaces.  I know typically we use tabs...but since npm automatically formats with spaces the next install would change this.

Comment (cwebley) [inline package.json]: fixed

**Comment by wesleytodd** [comment]
Force push those up as a single commit and I will merge.  Or rebase or whatever suits your fancy :)

**Comment by cwebley** [comment]
rebased and squashed.

**Pull request merged**